<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vagregapasajero</class>
 <widget class="QDialog" name="vagregapasajero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>117</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Pasajero</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>271</width>
     <height>380</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txfNombre"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Apellidos:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txfApellidos"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txfID"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txfTelefono"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Nacionalidad:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="txfNacio"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="txfPeso"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Estatura:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="txfEstatura"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QComboBox" name="cbxDestino">
      <item>
       <property name="text">
        <string>San Jose</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alajuela</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cartago</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heredia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Guanacaste</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Puntarenas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limon</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QPushButton" name="btnAgregar">
      <property name="text">
       <string>Agregar Pasajero</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="btnAgregarMaleta">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Agregar Maleta</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="2">
     <widget class="QPushButton" name="btnVolver">
      <property name="text">
       <string>Listo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>btnAgregar</zorder>
  <zorder>btnAgregarMaleta</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
